### PR TITLE
Adds output silencing option

### PR DESCRIPTION
**Pull request by johnnyman727**
This is useful for consumers of the API exposed by the CLI (in an upcoming PR) because if you're using it programmatically, you likely don't want to clutter your output with all the things the CLI reports.

Part of the disintegration of #555 and needed for the test rig.

### CODE REVIEW

File: lib/controller.js [hunk 1]
```diff
@@ -24,22 +24,40 @@ var responses = {
...
+controller.defaultHelpers = function(opts, next) {
+  // First disable output if necessary
+  return controller.outputHelper(opts)
+    // Then make sure our SSH Key path is updated
+    .then(() => controller.keyHelper(opts))
+    // before continuing to the next function
+    .then(() => next(opts));
+};
+
+controller.keyHelper = function(opts) {
```
Comment (rwaldron) [inline lib/controller.js]: Seems unrelated.

Comment (johnnyman727) [inline lib/controller.js]: Since there are two functions every command should call (`keyHelper` and `outputHelper`), I decided it made sense to just refactor into a single function that would be easier to build upon.

Comment (rwaldron) [inline lib/controller.js]: Sure, but I mean that it's unrelated to the PR's title :P "Adds output silencing option"?

Comment (johnnyman727) [inline lib/controller.js]: It was necessary because I wanted to add the function below (`outputHelper`) without making the code look too hairy. `keyHelper` originally set the default SSH key and then was responsible for calling the subsequent command (either `list` or `get`) but I figured it makes more sense for one function to call each of the helpers and then pass it on to the command.
